index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Code Viewer</title>
+</head>
+<body>
```

Comment:
And please don't forget about Linters

Reply:
- [ ] Kindly I recommend you to phrased the comments politely, as suggestions instead of orders and includes an explanation, a "why", for each requested change with comments are clear and easy to understand, Kindly mention why the setup linters for your project is important .